web/src/components/layout/AppSidebar.vue

```
@@ -0,0 +1,31 @@
+<script setup lang="ts">
+import { LayoutDashboard, Map, Newspaper } from 'lucide-vue-next'
```

Biome (2.3.14)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)